docs/docs/handbook/student_management.md

```diff
@@ -60,6 +61,36 @@ To create new students in your tutorial click the "+ New" button on the upper ri
 
 1.  **Abort button**: Closes the form without saving the information.
 
+## Import Students
+
+To import students from a CSV file click on the "<IconInText icon={GenerateIcon} /> Generate" button in the upper right. A step-by-step wizard will guide you through the import process.
+
+1. **Import CSV content**: Import the CSV content by either copy & paste the content in the input field or by uploading the file. You can upload the file by clicking the "Upload CSV-file" button and the content will be pasted into the input field after uploading. If the separator is not correctly detected on the next page you can change the seperator on _this_ page using the "Separator" input in the upper right.
+
+   :::tip Drag & Drop
+   You can also drag & drop the file on the button
+   :::
+
+1. **Map Columns**: The TMS tries to find all needed columns automatically. If the columns are not properly detected you can change them through the dropdown menus on this page. The following columns are supported (_required_ ones must be provided):
+
+   - _Firstname (**required**)_: The firstname of the student.
+
+   - _Lastname (**required**)_: The lastname of the student.
+
+   - _Status_: The status of the student, one of `ACTIVE`, `INACTIVE`, and `NO_SCHEIN_REQUIRED`. Matches the provided value case-insensitive. If no value is provided, the status is set to `ACTIVE`.
```

Comment:
What happens if an invalid value is provided? Mabye change the text to "if no _or an invalid_ value is provided, ...".

Reply:
I actually thought this would make the import fail, however, one look in the source code reveals that it also results in active.
Therefore adopted docs like you recommended with db6145dddca6f259a6ea2655926287978e6b5a1a